src/models/Factor.js

```diff
@@ -105,7 +107,7 @@ function (Okta, Q, factorUtil, Util, Errors, BaseLoginModel) {
       factorLabel: {
         deps: ['provider', 'factorType', 'vendorName'],
         fn: function (provider, factorType, vendorName) {
-          if (_.contains(['DEL_OATH', 'GENERIC_SAML', 'GENERIC_OIDC'], provider)) {
+          if (_.contains(['DEL_OATH', 'GENERIC_SAML', 'GENERIC_OIDC', 'CUSTOM'], provider)) {
```

Comment:
Will this be changed as part of this PR?

Reply:
wil take care in enroll PR. Not sure if I would have to modify this.

Reply:
wil take care in enroll PR. Not sure if I would have to modify this.